sci-libs/cddlib/cddlib-094m-r1.ebuild

```diff
@@ -14,7 +14,7 @@ SRC_URI="https://github.com/${PN}/${PN}/releases/download/${MY_PV}/${MY_P}.tar.g
 
 SLOT="0"
 LICENSE="GPL-2+"
-KEYWORDS="amd64 ~arm ~ppc x86 ~amd64-linux ~x86-linux"
+KEYWORDS="amd64 ~arm ppc ~riscv x86 ~amd64-linux ~x86-linux"
```

Comment:
why ppc?

Reply:
Oops, probabily it is merge conflict from git rebase. I will fix that.